diff.js

```diff
@@ -454,7 +437,6 @@
           curRange.push.apply(curRange, map(lines, function(entry) {
             return (current.added ? '+' : '-') + entry;
           }));
-          eofNL(curRange, i, current);
```

Comment:
I had trouble adapting its logic to the hunk'd format. Since trying to add no-nl each time we have an add/delete entry seemed inefficient, I opted for a rewrite.